arbitrum-docs/for-devs/dev-tools-and-resources/chains-info.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Chains information Arbitrum'
+sidebar_label: 'Chains info'
+description: 'Chains information Arbitrum'
+author: anegg0
+sme: anegg0
+user_story: As a current or prospective Arbitrum developer, I need to find endpoints and other information about the Arbitrum network so that I can interact with them.
```

Comment:
```suggestion
user_story: As a current or prospective Arbitrum developer, I need to find endpoints and other information about the Arbitrum chains so that I can interact with them.
```